### PR TITLE
uniform the variable names in the documentation

### LINKED ISSUE #127: [JOSS] Minor docs code snippet tweaks for consistency
In the "Splitting by Mask" docs section, the code snippets use `ds_valid` and `train_test_split` where the rest of the examples are `ds` and `motrainer.train_test_split`, so I'd change those just so it's consistent throughout

https://github.com/VegeWaterDynamics/motrainer/blob/f74acd8c44cc0be7539095c8a6ab4c0c1560f6e4/docs/usage_split.md?plain=1#L184-L194

Instead would be:

```python 
 mask = ds["time"] < np.datetime64("2017-01-01") 
 train, test = motrainer.train_test_split(ds, mask=mask) 
 ``` 

 ```python 
 mask = ds["time"] < np.datetime64("2017-01-01") 
 train, test = motrainer.train_test_split(ds, mask=mask, reverse=True) 
 ```

### PR DESCRIPTION
**Pull request by rogerkuou**
fix #127

### CODE REVIEW

**Comment by rogerkuou** [comment]
Hi @SarahAlidoost, this is the second PR for the JOSS review. Thanks!

**Review by SarahAlidoost — APPROVED** [review]
@rogerkuou thanks for the fixes :+1:
